config/classifiers.yml

```diff
@@ -105,6 +112,7 @@ classifiers:
           stale_branches_count: 1
           life_period: 1
           last_changed: 1
+          questions_resolved_percent: 0.5
```

Comment:
Ok, I moved the metric to popularity with weight 1 (but inclined to decrease it). 
Also I will add both metrics to the maturity section as it directly affects one of the key aspect of software maturity - activity of the community, with weights 1 and 0.5 for total and resolved percent respectively. What do you think about this scheme?